### ISSUE TITLE
Add a method to access the current Network's SSID 

### ISSUE THREAD

**Issue by Audrey-Ann** [aboozaid/react-native-tethering#25]
Hello there! 

Thanks for providing this package! 

I was wondering if it'd be possible to include a method which would return the current connected network's SSID? Like: 

```
getCurrentWifiSSID
```

which would return the SSID or null (if not connected)

for the `@react-native-tethering/wifi` package

??

After reading online it seems possible if the device grants access to at least the foreground location (?)

Do you think it would possible to add this method and/or do you see any issues with this new request?

Thank you in advance for your help and support :)

**Comment by aboozaid** [comment]
I thought you could handle this from react-native side when a connection occur you could store that SSID and remove it once disconnected. I may add it in the next versions but I'm not sure if that still possible with new android versions which break many many things.

**Comment by Audrey-Ann** [comment]
Indeed, I was using a library called `@react-native-community/netinfo` to get the state of the network, however I was hopping that the `react-native-tethering` would include it as well 🙏
